app/main.py

```diff
@@ -1 +1,63 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if not isinstance(command, str):
+        print("Incorrect format")
+        return
+
+    cmd_list = command.strip().split(" ")
+
+    if len(cmd_list) and cmd_list[0] != "mv":
```

Comment:
The condition should check if the first element of cmd_list is 'mv'. The current condition will not work correctly if cmd_list is empty. Consider changing the condition to `if not cmd_list or cmd_list[0] != 'mv':` to ensure it handles empty lists properly.